Directory.Build.props

```diff
@@ -24,8 +24,8 @@
     <DefaultTargetPlatformMinVersion>15063</DefaultTargetPlatformMinVersion>
 	  <PackageOutputPath>$(MSBuildThisFileDirectory)bin\nupkg</PackageOutputPath>
 	  
-	  <SignAssembly>true</SignAssembly>
-    <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)toolkit.snk</AssemblyOriginatorKeyFile>
+	  <SignAssembly Condition="'$(SignAssembly)' == ''">true</SignAssembly>
```

Comment:
Do we still need this if importing the common props makes it do the right thing?
My concern is about other things that might be affected too.

Reply:
It's a good bet this doesn't work like we think it should; however, in cases where we may have issue for Win32 we are already guarding sets with conditions: https://github.com/rjmurillo/UWPCommunityToolkit/blob/0c11e16900af52118f73e2eece9109e14f5f619a/Directory.Build.props#L34-L37
Cases where we don't guard, we're okay with overwriting what has been provided

Reply:
I was mostly thinking about this from #2299 and other contributions like that in the future, where we want to coerce the behavior. We now know that a csproj should be reviewed for common props imports to ensure nothing else wacky goes on, but with this change a csproj is not required to be reviewed. What's more it is no longer a requirement to manually import those props (I'm not sure how they were missed in my .csproj), and it is no longer a requirement to  convert to SDK-style projects, which is not the default in WinForms/WPF.

Reply:
I'll leave that to @nmetulev to weigh in. We're converting the Win32 projects to be SDK-style anyway, so this won't be an issue anymore. I think it should be a requirement that the csproj's use the SDK-style for this project unless there's extenuating circumstances why it won't work (and then I can try to fix it 😀)

Reply:
We are not converting the tests to SDK style project, so I think this is still helpful. And I like that it gives us a redundancy

Reply:
Why aren't we converting them to SDK style? There's no reason not to.

Reply:
If it's possible, I'm ok with converting them, could use your help there

Reply:
Okay. It's all possible, basically the same as any other project. In this case, it requires an extra import at the bottom for the `  <Import Project="$(VSToolsPath)\TeamTest\Microsoft.TestTools.targets" Condition="Exists('$(VSToolsPath)\TeamTest\Microsoft.TestTools.targets')" />`, but it's the same otherwise.

Did we ever have a discussion around what test platform to use? They're currently using `mstest`, but most of the other OSS/Microsoft projects are all using xUnit.